## doxygen-main-link.diff by Jens Peter Secher <jps@debian.org>
##
## Exclude links, either because they do not exists or because
## Doxygen fails to recognise them.

Index: libcrypto++/cryptlib.h
===================================================================
--- libcrypto++.orig/cryptlib.h	2010-08-09 20:42:43.000000000 +0200
+++ libcrypto++/cryptlib.h	2010-08-09 20:43:01.000000000 +0200
@@ -71,7 +71,6 @@
 </dl>
 
 <p>This reference manual is a work in progress. Some classes are still lacking detailed descriptions.
-<p>Click <a href="CryptoPPRef.zip">here</a> to download a zip archive containing this manual.
 <p>Thanks to Ryan Phillips for providing the Doxygen configuration file
 and getting me started with this manual.
 */
